prometheus-server-metrics/src/main/scala/org/http4s/server/prometheus/PrometheusMetrics.scala

```diff
@@ -213,22 +239,24 @@ object PrometheusMetrics {
   def apply[F[_]: Sync](
       c: CollectorRegistry,
       prefix: String = "org_http4s_server",
+      tag: Option[String] = Option.empty[String],
```

Comment:
Why just one tag?  Why must it be called `"tag"`?  Can you take  `Map[String, String]`?